tutorials/mandels_problem.ipynb

```diff
@@ -44,8 +44,8 @@
     "import porepy as pp\n",
     "import sys\n",
     "\n",
-    "sys.path.append(\"..\")\n",
```

Comment:
I would like us to use pathlib.

Reply:
@IvarStefansson I figured out that the original solution `from examples import mandel_biot` didn't work for me, because I had another python module `examples` in my PATH. So this was my personal problem, sorry for the confusion. I will change it back as it was. I massively agree with using pathlib.

Reply:
I also agree with using pathlib, seems the most elegant (and safe?) option.

Reply:
Done